### ISSUE TITLE
Prevent the use of an externa jquery script

### ISSUE THREAD

**Issue by it0056** [janizde/WP-Opening-Hours#53]
I customized my version of your script to use the jquery core instead because of: 
- Better load performance
- more native WP
- Other plugins will not run into problems caused by this plugin.

**File: Classes\OpeningHours\OpeningHours.php**
**Replace the following code (line 98 to 119):**
```
public function loadResources () {
    wp_register_script(
      self::PREFIX . 'js',
      plugins_url('dist/scripts/main.js', op_bootstrap_file()),
      array('jquery', 'jquery-ui'),
      self::VERSION,
      true
    );

    wp_register_style(
      self::PREFIX . 'css',
      plugins_url('dist/styles/main.css', op_bootstrap_file())
    );

    // Backend Styles and Scripts
    wp_enqueue_script('jquery-ui');

    if (!wp_script_is('jquery-ui')) :
      wp_register_script('jquery-ui', '//code.jquery.com/ui/1.11.4/jquery-ui.min.js', array('jquery'));
      wp_enqueue_script('jquery-ui');
    endif;

```
**With the code:**
```
public function loadResources () {
    wp_register_script(
      self::PREFIX . 'js',
      plugins_url('dist/scripts/main.js', op_bootstrap_file()),
      array('jquery'),
      self::VERSION,
      true
    );

    wp_register_style(
      self::PREFIX . 'css',
      plugins_url('dist/styles/main.css', op_bootstrap_file())
    );

    // Backend Styles and Scripts
    wp_enqueue_script('jquery-ui-core');
    wp_enqueue_script('jquery-ui-datepicker');

```

Please keep up the good work, love your plugin!

**Comment by janizde** [comment]
thank you, I've added this and it will be released with 2.1

**Comment by janizde** [comment]
released in v2.1
